### PR TITLE
Updated my portfolio link

### PR DESCRIPTION
**Pull request by Chamberezigbo**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4905"></a>

### CODE REVIEW

**Comment by ChinmayMhatre** [comment]
please remove the file that is in /public/data

**Comment by ChinmayMhatre** [comment]
use this website to verify your json file https://jsonlint.com

File: data/chamberezigbo.json [hunk 1]
```diff
@@ -1,59 +1,79 @@
...
+>>>>>>> 444abc75fa64b328ee08acf24eca5a99d01d3a77:data/chamberezigbo.json
+  "avatar": "https://github.com/chamberezigbo.png",
   "links": [
     {
-      "name": "My portfiolo site",
-      "url": "https://shiny-starburst-6f2c53.netlify.app",
-      "icon": "FaLink"
+      "name": "My portfolio site",
+      "url": "https://wondrous-churros-f6aea2.netlify.app",
+      "icon": "link"
```
Comment (ChinmayMhatre) [inline data/chamberezigbo.json:14]: Please refer these docs and change the icons for all. https://linkfree.eddiehub.io/icons

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Review by Chamberezigbo — COMMENTED** [review]
icon names resolved

File: data/chamberezigbo.json [hunk 2]
```diff
@@ -1,59 +1,79 @@
 {
   "name": "Chamberlain Ezigbo",
   "type": "personal",
+<<<<<<< HEAD:public/data/chamberezigbo.json
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:4]: This is causing the json check fail which should be removed. 

```suggestion
```

File: data/chamberezigbo.json [hunk 3]
```diff
@@ -1,59 +1,79 @@
...
   "bio": "JavaScript Engineer | Front-end Developer",
-
+=======
+  "bio": "JavaScript Engineer | Full-stack Developer",
+>>>>>>> 444abc75fa64b328ee08acf24eca5a99d01d3a77:data/chamberezigbo.json
+  "avatar": "https://github.com/chamberezigbo.png",
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:9]: Bio is added twice, one should be removed. Also, we no longer need the `avatar` key. 

```suggestion
```

File: data/chamberezigbo.json (hunk 1 above)
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:14]: Invalid icon is used. It should be `FaLink`

```suggestion
      "icon": "FaLink"
```

File: data/chamberezigbo.json [hunk 4]
```diff
@@ -1,59 +1,79 @@
...
-      "icon": "FaLink"
+      "name": "My portfolio site",
+      "url": "https://wondrous-churros-f6aea2.netlify.app",
+      "icon": "link"
     },
     {
       "name": "Follow me on GitHub",
       "url": "https://github.com/chamberezigbo",
-      "icon": "FaGithub"
+      "icon": "github"
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:19]:
```suggestion
      "icon": "FaGithub"
```

File: data/chamberezigbo.json [hunk 5]
```diff
@@ -1,59 +1,79 @@
...
     },
     {
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/Chamberlin5050",
-      "icon": "FaTwitter"
+      "icon": "twitter"
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:24]:
```suggestion
      "icon": "FaTwitter"
```

File: data/chamberezigbo.json [hunk 6]
```diff
@@ -1,59 +1,79 @@
...
     },
     {
       "name": "Follow me on LinkedIn",
       "url": "https://linkedin.com/in/chambertech",
-      "icon": "FaLinkedin"
+      "icon": "linkedin"
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:29]:
```suggestion
      "icon": "FaLinkedin"
```

File: data/chamberezigbo.json [hunk 7]
```diff
@@ -1,59 +1,79 @@
...
+      "icon": "discord",
+      "color": "#3a3240",
+      "description": "Use the NodeJS release of this library to validate your forms server-side.",
+      "url": "https://github.com/Octagon-simon/octaValidate"
+    },
     {
       "title": "Developed and maintained website for charity gate",
       "date": "May 2022",
-      "icon": "FaDiscord",
+      "icon": "discord",
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:44]:
```suggestion
      "icon": "FaDiscord",
```

File: data/chamberezigbo.json [hunk 8]
```diff
@@ -1,59 +1,79 @@
...
       "color": "#3a3240",
       "description": "Developed a website for a non governmental organization ",
       "url": "http://charitygatefoundation.com/"
     },
     {
+      "title": "Tailor Skit",
+      "date": "december 2022",
+      "icon": "github",
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:52]:
```suggestion
      "icon": "FaGithub",
```

File: data/chamberezigbo.json [hunk 9]
```diff
@@ -1,59 +1,79 @@
...
       "url": "https://linkedin.com/in/chambertech",
-      "icon": "FaLinkedin"
+      "icon": "linkedin"
     }
   ],
   "milestones": [
+         {
+      "title": "This JavaScript library helps to validate your frontend (HTML) forms using validation rules, sophisticated regular expressions and form input attributes.",
+      "date": "May 2022",
+      "icon": "discord",
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:36]:
```suggestion
      "icon": "FaDiscord",
```

File: data/chamberezigbo.json [hunk 10]
```diff
@@ -1,59 +1,79 @@
...
+      "icon": "github",
+      "color": "blue",
+      "description": "Tailor's kit is a large database & productivity software that helps Tailors to save, manage, access client's data, and track client's orders from anywhere and at anytime with ease.",
+      "url": "https://tailorskit.com/"
+    },
+     {
       "title": "Started working for a IG Hub as intern",
       "date": "April 2021",
-      "icon": "FaGithub",
+      "icon": "github",
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:60]:
```suggestion
      "icon": "FaGithub",
```

File: data/chamberezigbo.json [hunk 11]
```diff
@@ -1,59 +1,79 @@
...
       "color": "blue",
       "description": "I maintained already existing code and together with other interns, we built sites from scratch for the company.",
       "url": "https://www.github.com/chamberezigbo"
     },
     {
       "title": "Started working for a Rad5 as intern",
       "date": "April 2020",
-      "icon": "FaGithub",
+      "icon": "github",
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:68]:
```suggestion
      "icon": "FaGithub",
```

File: data/chamberezigbo.json [hunk 12]
```diff
@@ -1,59 +1,79 @@
...
       "color": "blue",
       "description": "Collaborated with fellow programmers and designers to build and review new and existing code during the internship",
       "url": "https://www.rad5.com"
     },
     {
       "title": "Learning to Code!",
       "date": "April 2019",
-      "icon": "FaGithub",
+      "icon": "github",
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:76]:
```suggestion
      "icon": "FaGithub",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey @Chamberezigbo, a few changes are required. 

- Rename your file from `chamberezigbo.json`  - >  `Chamberezigbo.json` (`C` should be capital as in your username).
- Remove the other file in `public/data` directory.
- Make the changes below as suggested. 

Let us know if you need any help with this.

Or 

Let us know if you want us to make these changes.

**Comment by priyankarpal** [comment]
I wanna say that if you're facing any trouble you can close this PR and open a new fresh PR. Don't forget to check the official docs. https://linkfree.eddiehub.io/docs/quickstart

**Pull request closed without merging**